python/tests/integration/arcticdb/test_arctic.py

```diff
@@ -717,6 +717,33 @@ def test_write_batch_pickle_duplicate_symbols(arctic_library):
     assert not lib.list_symbols()
 
 
+def test_write_batch(arctic_library):
+    """Should be able to write different size of batch of data."""
+    lib = arctic_library
+    lib._nvs._lib_cfg.lib_desc.version.write_options.segment_row_size = 10
```

Comment:
This is an option you can control at library creation time. Please add a fixture for creating libraries that accepts a `LibraryOptions` parameter.

Reply:
done as discussed